pkg/cmd/prebuild/add.go

```diff
@@ -21,11 +21,30 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	runOnAddFlag bool
+)
+
 var prebuildAddCmd = &cobra.Command{
-	Use:     "add",
+	Use:     "add [flags]",
 	Short:   "Add a prebuild configuration",
 	Args:    cobra.NoArgs,
 	Aliases: []string{"new", "create"},
+	Long: `
```

Comment:
No need for this long description. This is auto-generated by cobra when running `--help`

Reply:
Oh ok removed the long description